docs/book/countries.rst

```diff
@@ -6,62 +6,62 @@ Countries
 
 **Countries** are a part of the :doc:`Addressing </book/addresses>` concept.
 The **Country** entity represents a real country that your shop is willing to sell its goods in (for example the UK).
-It has an ISO code to be identified easily (ISO 3166-1 alpha-2).
+It has an ISO code to be identified easily (`ISO 3166-1 alpha-2 <http://www.iso.org/iso/country_codes>`_).
 
 Countries might also have **Provinces**, which is in fact a general name for an administrative division, within a country.
 Therefore we understand provinces as states of the USA, voivodeships of Poland, cantons of Belgium or bundesländer of Germany.
 
 How to add a country?
 ---------------------
 
-To give you a better insight into Countries, let's have a look how to prepare and add a Country to the system programatically.
-We will do it together with a province at once.
+To give you a better insight into Countries, let's have a look on how to prepare and add a Country to the system programmatically.
+We will do it with a province at once.
 
-Firstly you will need factories for countries and for provinces. Create new objects with them.
-To the newly created objects assign codes. The provinces are added to a country as a collection.
-Finally using a repository for countries - add the country to the system.
-From now on it will be available to use.
+You will need factories for countries and for provinces for creating them:
```

Comment:
There is three times a "for" word in the statement. Could it be created with "in order to" or something other?